cookbook/deployment/tools.rst

```diff
@@ -66,9 +66,10 @@ Using Build Scripts and other Tools
 There are also tools to help ease the pain of deployment. Some of them have been
 specifically tailored to the requirements of Symfony.
 
-`Capifony`_
-    This Ruby-based tool provides a specialized set of tools on top of
-    `Capistrano`_, tailored specifically to Symfony projects.
+`Capistrano`_/`symphony`_
+    `Capistrano`_ is a remote server automation and deployment tool written in Ruby. 
+    `symphony`_ is a plugin to ease symphony related tasks, inspired by `Capifony`_
+    (works only with Capistrano 2 )
```

Comment:
this may be confusing IMO, as readers may think it relate to the whole paragraph rather than to Capifony